src/app/pages/loading-states/loading-states.component.html

```diff
@@ -0,0 +1,54 @@
+<mat-toolbar color="primary" flexLayout="row">
+    <button *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px;">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <h6>Loading Example</h6>
+    <div fxFlex></div>
+    <button mat-button color="white" (click)="refreshData()">
+        <mat-icon>autorenew</mat-icon>
+    </button>
+</mat-toolbar>
+
+<div style="padding: 20px;">
+    <mat-grid-list [cols]="cols" gutterSize="12px" rowHeight="270px">
+        <mat-grid-tile *ngFor="let device of deviceList">
+            <mat-card class="padding0">
+                <app-placeholder-card *ngIf="!device.data" id="placeholder"></app-placeholder-card>
+
+                <mat-list class="padding0" *ngIf="device.data" id="data-list">
+                    <pxb-hero-banner [divider]="true">
+                        <pxb-hero label="Grade" value="{{ device.data.heroValue }}" units="/100" size="36">
+                            <span [style.color]="interpreter.interpretGrade(device.data.heroValue)" pxb-primary>
+                                {{ interpreter.interpretIcon(device.data.heroValue) }}
+                            </span>
+                        </pxb-hero>
+                        <pxb-hero label="Load">
+                            <pie-progress
+                                [style.color]="interpreter.interpretColor(100 - device.data.loadValue)"
+                                percent="{{ device.data.loadValue }}"
+                                size="36"
+                                pxb-primary
+                            ></pie-progress>
+                            <pxb-channel-value value="{{ device.data.loadValue }}" units="%"> </pxb-channel-value>
+                        </pxb-hero>
+                        <pxb-hero label="Battery" value="{{ device.data.battery }}" units="%">
+                            <battery-progress
+                                [style.color]="interpreter.interpretColor(device.data.battery)"
+                                percent="{{ device.data.battery }}"
```

Comment:
strange these are still showing 100% full when they load. The Pie works as expected with the battery data, so this might be a problem with the progress icon itself.

Reply:
We've got a separate issue to fix this in the hero now